releasenotes/notes/detect-win-ec2-instance-3e90f6ad7ae03683.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    An hostname starting with ``EC2AMAZ-`` has its canonical hostname set to the instance id.
```

```suggestion
    Hostnames starting with ``EC2AMAZ-`` will now have its canonical hostname set as instance id.
    You might need to update these hostnames in your dashboards and alarms accordingly.
```

It's good to provide some actionable upgrade path to make the 'upgrade' notes less scary.